src/view/com/lists/ProfileLists.tsx

```diff
@@ -75,7 +75,12 @@ export function ProfileLists({
       items = items.concat([EMPTY])
     } else if (data?.pages) {
       for (const page of data?.pages) {
-        items = items.concat(page.lists)
+        items = items.concat(
```

Comment:
Do we have other places needing a similar fix?

Reply:
Haven't noticed any yet, just grouped these fixes in a pr for now